python/drned_xmnr/op/common_op.py

```diff
@@ -72,7 +72,6 @@ def cli_filter(self, msg):
     def perform(self):
         result, _ = self.devcli_run('load-default-config.py', [])
         if result != 0:
-            self.log.debug("Exception: " + repr(e))
```

Comment:
a side bugfix for previous new action commit - same invocation pattern as the current PR